src/components/AdminForm.tsx

```diff
@@ -7,20 +7,28 @@ const AdminForm: React.FC = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [isSuperAdmin, setIsSuperAdmin] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(""); // State to hold the error message
```

Comment:
This comment is not very useful, the code is clear and simple enough to document itself

Reply:
done